package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-dates",
-  "version": "21.8.0",
+  "version": "21.9.0",
```

Comment:
PRs shouldn't ever bump versions; I'll revert this when rebasing.

Reply:
Sorry about that. I didn't know. Thanks for the feedback!